### ISSUE TITLE
The training timer of version 0.10.1 has a bug

### ISSUE THREAD

**Issue by ThunderbirdWang** [jdb78/pytorch-forecasting#980]
- PyTorch-Forecasting version:0.10.1
- PyTorch version:1.11.0+cu113
- Python version:3.8.10
- Operating System:Tensorflow docker for latest-gpu-jupyter on Ubuntu 20.04.1

The Epoch 0 of training is expected to take about 2 hours, that's right：
Epoch 0:   1%|          | 76/7191 [**01:24<2:11:35**,  1.11s/it, loss=5.24e+03, v_num=15, train_loss_step=4.93e+3]

It's been wrong since the Epoch 1:
Epoch 1:   1%|          | 59/7191 [**2:03:52<249:33:47**, 125.97s/it, loss=44.5, v_num=15, train_loss_step=46.70, val_loss=64.90, train_loss_epoch=447.0]

Epoch 2:   1%|          | 48/7191 [**4:05:55<609:56:12**, 307.40s/it, loss=29.1, v_num=15, train_loss_step=30.30, val_loss=44.30, train_loss_epoch=35.70]

Epoch 3:   1%|          | 81/7191 [**6:08:46<539:29:44**, 273.16s/it, loss=22.2, v_num=15, train_loss_step=23.80, val_loss=48.00, train_loss_epoch=25.70]
